fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -224,10 +224,11 @@ public class OperationType {
     public static final short OP_SAVE_TRANSACTION_ID = 104;
     public static final short OP_SAVE_AUTO_INCREMENT_ID = 105;
     public static final short OP_DELETE_AUTO_INCREMENT_ID = 106;
-
     // light schema change for add and drop columns
     public static final short OP_MODIFY_TABLE_ADD_OR_DROP_COLUMNS = 107;
 
+    public static final short OP_UPSERT_TRANSACTION_STATE_BATCH = 108;
```

Comment:
how old version sr read the new editlog when rollback

Reply:
There is a compatibility issue exactly，if upgrade at first and then downgrade